mapviz_plugins/src/measuring_plugin.cpp

```diff
@@ -36,11 +36,7 @@
 #include <QTextStream>
 #include <QPainter>
 
-#if QT_VERSION >= 0x050000
```

Comment:
Are these unneeded in ros2?

Reply:
Yep, ROS2 (or more specifically, the distros ROS2 runs on) only supports Qt 5, so support for lower versions is just wasting space.